zstacklib/zstacklib/utils/upload_task.py

```
@@ -0,0 +1,330 @@
+    while True:
+        b = entity.readline()
+        if not b:
+            return
+
+        b = b.strip()
+        if b == ib:
+            break
+
+    return ib
+
+def stream_body(entity, boundary, param, task, ioctx):
```

Ruff (0.13.1)

Unused function argument: `ioctx`

(ARG001)